GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -568,6 +568,9 @@ private bool NeedsGitStatusMonitor()
 
         private void UICommands_PostRepositoryChanged(object sender, GitUIEventArgs e)
         {
+            // Process events e.g. from closing FormProcess, particularly after rebase / conflict resolution actions
```

Comment:
Why not doing this in `RefreshRevisions()`?
I would like to replace all internal invocations of RepoChanged in FormBrowse in #9888

Reply:
Just too many functions with similar names.

Reply:
The handling should be changed to call RefreshRevisions by default. Should this be moved then?

Reply:
> Just too many functions with similar names.

I just wanted to express that I didn't notice in the diff that it was added to a different function.
I inserted it before some refresh call and was happy it worked.